docs/configuration.md

```diff
@@ -34,6 +34,26 @@ This command actually uses [the template file](../templates/cluster-template.yam
 
 Note: You can use [the template file](../templates/cluster-template.yaml) by manually replacing values.
 
+**Note:** By default the command creates highly available control plane with internal OpenStack cloud provider. If you wish to create highly available control plane with external OpenStack cloud provider or single control plane without load balancer, use external-cloud-provider or without-lb [flavor](https://cluster-api.sigs.k8s.io/clusterctl/commands/config-cluster.html#flavors) respectively.
+
+```bash
+# Using 'external-cloud-provider' flavor
+clusterctl config cluster capi-quickstart \
+  --flavor external-cloud-provider
+  --kubernetes-version v1.18.2 \
+  --control-plane-machine-count=1 \
```

Comment:
control-plane-machine-count should be 3.
because we say "create highly available control plane with external OpenStack cloud provider" as above, and actually user would expect so.

Reply:
Updated